airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/common.py

```diff
@@ -326,13 +326,18 @@ async def get_step_result(self, container: Container) -> StepResult:
             content=container.file(self.REPORT_LOG_PATH),
             to_upload=True,
         )
+        # When the current connector is not certified we want to report the acceptance test results but not consider them in the overall status of the test pipeline
+        consider_result_in_overall_status = (
+            self.context.connector.metadata.get("supportLevel") == "certified" or self.context.ci_context == CIContext.MASTER
```

Comment:
@bnchrch this might change if we go for a different gate

Reply:
Can we pull this out into a helper method so it's easier to find?